requirements.txt

```diff
@@ -36,4 +36,5 @@ git+https://github.com/SatelliteQE/airgun.git@master#egg=airgun
 git+https://github.com/SatelliteQE/nailgun.git@master#egg=nailgun
 git+https://github.com/SatelliteQE/satellite6-upgrade.git@master#egg=satellite6-upgrade
 git+https://github.com/SatelliteQE/automation-tools@master#egg=automation-tools
+git+https://github.com/RonnyPfannschmidt/webdriver_kaifuku@master#egg=webdriver_kaifuku
```

Comment:
This should be an airgun dependency, as its not something we'll use directly in robottelo.

it can also be installed from pypi:
https://pypi.org/project/webdriver-kaifuku/